cmake/windows/resources/installer-Windows.iss.in

```diff
@@ -62,3 +74,27 @@ begin
     Result := InstallPath
 end;
 
+
+// Remove old OBS-NDI conflicting plugin - by Trouffman for DistroA https://github.com/DistroAV/DistroAV/
+
+// CAREFUL : this target the old AppID.
+const UninstallRegisterPath = 'Software\Microsoft\Windows\CurrentVersion\Uninstall\{A9039243-4FE7-45E7-8B11-7DC1ACB67B9D}_is1';
+
+function GetOldAppUninstallerPath(Param: String): String;
+var
+  UninstallerPathRegistry: String;
+// Param is required to be used in other sections.
+begin
+  result := '';
+  // Check primary registry location
+  if ( RegQueryStringValue(HKLM, UninstallRegisterPath, 'UninstallString', UninstallerPathRegistry) ) then
+      result := RemoveQuotes(UninstallerPathRegistry)
+  // Check alternative Registry location
+  else if ( RegQueryStringValue(HKCU, UninstallRegisterPath, 'UninstallString', UninstallerPathRegistry) ) then
+      result := RemoveQuotes(UninstallerPathRegistry);
+end;
+
+function UninstallOldAppAvailable(): Boolean;
+begin
+  if (GetOldAppUninstallerPath('') <> '') then Result := True else Result := False;
```

Comment:
I'm not sure why you are passing `''` as a Param to GetOldAppUninstallerPath and can't just hard code it inside GetOldAppUninstallerPath.

Reply:
Hehe this one drives me nuts.

I thought I commented in the PR message on the reason but here it is :

To call this function from {code:functionnamehere} it MUST take a single string argument. That argument can be empty.

As it is expecting the string. If you call it without the string. It fails. This is why it's there.

This requirement came from the pascal scripting doc in innosetup.

Took me some time to figure.

Reply:
Documentation for this specific question : https://jrsoftware.org/ishelp/index.php?topic=scriptintro

`The called function must have 1 String parameter named Param, and must return a String or a Boolean value[...]`